packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1267,6 +1267,12 @@ class ComboBoxInternal extends React.Component<IComboBoxInternalProps, IComboBox
   private _onLayerMounted = () => {
     this._onCalloutLayerMounted();
 
+    // need to call this again here to get the correct scroll parent dimensions
+    // when the callout is first opened
+    setTimeout(() => {
```

Comment:
nit, it's preferred to use the `Async` instance for timeouts
```suggestion
    this._async.setTimeout(() => {
```